admin/schema.php

```diff
@@ -884,6 +884,11 @@
 
 # Release marker: 1.3.0
 
+$g_upgrade[210] = array( 'AddColumnSQL', array( db_get_table( 'mantis_custom_field_table' ), "
+	color_background     C(8)		NOTNULL DEFAULT \" '' \",
```

Comment:
Why are NULL values not allowed?
I would use NULL / empty for not defined colors.

Reply:
changed. NULLs are allowed now.